resources/translations/fi.edn

```diff
@@ -272,7 +272,7 @@
                :logout "Kirjaudu ulos"
                :skip-navigation "Siirry sivun pääsisältöön"}
   :not-found-page {:not-found "Sivua ei löytynyt"
-                   :page-was-not-found "Tätä sivua ei löytynyt."}
+                   :page-was-not-found "Sivua ei löydy."}
```

Comment:
This should be matched to the heading. It can be shorter and the main explanation should be longer. See also English.

Reply:
Done.